src/RTClib.h

```diff
@@ -390,7 +396,8 @@ enum Pcf8523OffsetMode {
 /**************************************************************************/
 class RTC_PCF8523 {
 public:
-  boolean begin(void);
+  boolean begin();
+  boolean begin(TwoWire *wireInstance);
```

Comment:
change these to `TwoWire *wireInstance = &Wire` so there is only one begin() function

Reply:
thank you @ladyada 
I've been around the block a few times and still do silly things 😬

Reply:
thanks :)